### DISCUSSION TITLE
WorkFlow Dont works on V6.1.1

### DISCUSSION THREAD

**Discussion by Argonat62** [OpenIdentityPlatform/OpenIDM#62] [Q&A] [answered]
I try to run example.bpmn20.xml on a New OpenIDM System but I get the following Error:

`curl --header "X-OpenIDM-Username: openidm-admin" --header "X-OpenIDM-Password: openidm-admin" --request POST --header "Content-Type: application/json" --data '{"_key":"osgiProcess"}' "http://localhost:4080/openidm/workflow/processinstance?_action=create"|jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   228    0   206  100    22   1052    112 --:--:-- --:--:-- --:--:--  1169
{
  "code": 500,
  "reason": "Internal Server Error",
  "message": "Cannot invoke "org.forgerock.script.ScriptRegistry.takeScript(org.forgerock.script.ScriptName)" because "scriptRegistry" is null"
}`

Any Idea?

**Comment by maximthomas** [comment]
Hi @Argonat62, thanks for the issue! 
Could you please create an issue in the current repository and describe the environment and the detailed steps, so we could reproduce the problem? 
You can use this template [.github/ISSUE_TEMPLATE/bug_report.md](https://github.com/OpenIdentityPlatform/OpenIDM/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) as a base

**Comment by maximthomas** [comment, marked as answer]
The issue fixed in [6.1.2](https://github.com/OpenIdentityPlatform/OpenIDM/releases/tag/6.1.2) release
